src/config.rs

```diff
@@ -73,6 +73,34 @@ pub struct Config {
     #[serde(skip_serializing_if = "Option::is_none")]
     #[serde(default)]
     is_internal: Option<bool>,
+    /// This config option checks if internal authentication valid
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(default)]
+    is_internal_auth_valid: Option<String>,
+    /// This config option is path to acloud command.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(default)]
+    acloud_cmd_path: Option<String>,
+    /// This config option is path to acloud config.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(default)]
+    acloud_config_path: Option<String>,
+    /// This config option is a lunch target for each Android branch.
+    #[serde(skip_serializing_if = "HashMap::is_empty")]
+    #[serde(default)]
+    android_target: HashMap<String, String>,
+    /// This config option is a cheeps image name for ARCVM.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    #[serde(default)]
+    arc_vm_cheeps_image: Option<String>,
+    /// This config option is a betty image name for ARCVM.
+    #[serde(skip_serializing_if = "HashMap::is_empty")]
+    #[serde(default)]
+    arc_vm_betty_image: HashMap<String, String>,
+    /// This config option is a cheeps image name for ARC-container.
+    #[serde(skip_serializing_if = "HashMap::is_empty")]
+    #[serde(default)]
+    arc_container_cheeps_image: HashMap<String, String>,
 }
```

Comment:
I'm having some trouble understanding why a bunch of these need to be configs, aren't they alwyas passed at the command line? Could you write a brief explanation as to why they need to be included in config

Reply:
Because they are scripts to get image names and include confidential information.   
Also I have confirmed that the target name may be confidential information.

Reply:
Maybe it's a good idea to add some documentation in the code to explain so.
Also, even if the config values may be confidential, what kind of value format is expected should be documented as well. Could you add them? Thank you!

Reply:
@hikalium 
I already added the config option descriptions at https://github.com/google/lium/pull/201/commits/ee76923a205cc4f1e4c4e2fd6b4145bf895da068, so could you check them?